api/public/cerbos/policy/v1/policy.proto

```diff
@@ -230,7 +230,13 @@ message Match {
 }
 
 message Output {
-  string expr = 1;
+  message When {
+    string rule_activated = 1;
+    string rule_not_activated = 2;
```

Comment:
I think we should go for `rule_activated` and `condition_not_met`. The desired behaviour is for the `condition_not_met` output to be evaluated if everything except the condition matched, not if the rule wasn't activated because it is for a different role, but `rule_not_activated` would imply that the output would be produced even if the roles didn't match.